API.md

```diff
@@ -503,36 +783,132 @@ frisby.create('First test')
 ```
 
 ### only()
+
 Exclusively run this test. When `toss()` is invoked, the test is wrapped in a
 Mocha `describe.only` block instead of a `describe` block.
 
+### timeout(ms)
+
+Sets the timeout for this request.
+
+| Parameter | Description | Required |
+| --------- | ----------- | -------- |
+| ms | Integer. Sets the mocha timeout for this request. | Yes |
```

Comment:
Ah, this is incorrect. The `timeout()` is the timeout per request, which is compounded into a longer mocha timeout based on `delay()` and `retries()`. When the total time is reached, the test fails.

Reply:
Fixed.